src/Resources/Resources/help/New-AzADApplication.md

```diff
@@ -59,24 +59,33 @@ Creates a new azure active directory application. Below are the permissions need
 
 ## EXAMPLES
 
-### Example 1 - Create new AAD application.
+### Example 1: Create new AAD application.
 
-```
+```powershell
 PS C:\> New-AzADApplication -DisplayName "NewApplication" -HomePage "http://www.microsoft.com" -IdentifierUris "http://NewApplication"
 ```
 
 Creates a new azure active directory application without any credentials.
 
-### Example 2 - Create new AAD application with password.
+### Example 2: Create new AAD application with password.
 
-```
+```powershell
 PS C:\> $SecureStringPassword = ConvertTo-SecureString -String "password" -AsPlainText -Force
 PS C:\> New-AzADApplication -DisplayName "NewApplication" -HomePage "http://www.microsoft.com" -IdentifierUris "http:
 //NewApplication" -Password $SecureStringPassword
 ```
 
 Creates a new azure active directory application and associates password credentials with it.
 
+### Example 3
+
+Creates a new azure active directory application and associates password credentials with it. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+New-AzADApplication -DisplayName 'NewApplication' -HomePage 'https://www.microsoft.com' -IdentifierUris 'https://NewApplication' -Password <SecureString>
```

Comment:
We don't successfully detect the correct multi line command in example 2. It is broken inside  a string (http://NewApplication) so it doesn't need `